src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -284,3 +292,18 @@ you used the "--static-etcd-volume" flag), the underlying volume will not be
 removed.`)
 	return []*cobra.Command{deploy, undeploy}
 }
+
+func getDefaultDashImage() string {
+	// Try to get the latest according to our pachctl version, or fall back to the encoded default
+	url := fmt.Sprintf("https://pachyderm.io/versions/dash/%v.%v/latest", version.MajorVersion, version.MinorVersion)
```

Comment:
This really doesn't seem that much more convenient to me than just passing an argument to pachctl. It also seems like it could be really confusing if you're trying to deploy a new version of the UI and accidentally don't because our site was down that second. It just seems like it's creating a weird coupling between our release / deployment process and our website that doesn't need to be there. If we really want this behavior we could in theory get it by doing `--dash-image $(curl https://pachyderm.io/versions/...)` then at least if there's an error it'll be obvious what's going on.